pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -570,7 +570,7 @@ async def main():
                 output_toolset.max_retries = self._max_result_retries
                 output_toolset.output_validators = output_validators
         toolset = self._get_toolset(output_toolset=output_toolset, additional_toolsets=toolsets)
-        tool_manager = ToolManager[AgentDepsT](toolset)
+        tool_manager = ToolManager[AgentDepsT](toolset, default_max_retries=self._max_tool_retries)
```

Comment:
Can you please add a test that shows that the retries count passed to the agent is now respected?